src/lib/storage/storage_manager.hpp

```diff
@@ -1,5 +1,6 @@
 #pragma once
 
+#include <unordered_map>
```

Comment:
I dont see this used in the .hpp or .cpp - can it be removed?

Reply:
It's actually used on line 43. Note that, in theory, this import is not necessary, but we had some issues with a MacOS setup where the compiler did not recognize the `unordered_map` by default, so we had to include manually